saleor/attribute/migrations/0038_remove_assignedproductattribute.py

```diff
@@ -0,0 +1,108 @@
+# Generated by Django 3.2.20 on 2023-07-11 16:10
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("attribute", "0037_remove_assignedpageattribute"),
+    ]
+
+    state_operations = [
+        migrations.RemoveField(
+            model_name="attributeproduct",
+            name="assigned_products",
+        ),
+        migrations.RemoveField(
+            model_name="assignedproductattributevalue",
+            name="assignment",
+        ),
+        migrations.DeleteModel(
+            name="AssignedProductAttribute",
+        ),
+        migrations.AlterUniqueTogether(
+            name="assignedproductattributevalue",
+            unique_together={("value", "product")},
+        ),
+        migrations.AlterField(
+            model_name="assignedproductattributevalue",
+            name="product",
+            field=models.ForeignKey(
+                db_index=False,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="attributevalues",
+                to="product.product",
+            ),
+        ),
+    ]
+
+    operations = [
+        # fill in data that was previously done in a celery task
+        migrations.RunSQL(
+            """
+            UPDATE attribute_assignedproductattributevalue
+            SET product_id = (
+                SELECT product_id
+                FROM attribute_assignedproductattribute
+                WHERE attribute_assignedproductattributevalue.assignment_id = attribute_assignedproductattribute.id
+            )
+            WHERE id IN (
+                SELECT ID FROM attribute_assignedproductattributevalue
+                WHERE product_id IS NULL
+                ORDER BY ID DESC
+                FOR UPDATE
+            );
+            """,  # noqa
+        ),
```

Comment:
Missing `reverse_sql=migrations.RunSQL.noop` to ensure that migration is reversible